### PR TITLE
feat(surveys): Add disable config option

### PR DESCRIPTION
**Pull request by neilkakkar**
## Changes

Fixes https://github.com/PostHog/posthog-js/issues/932 
...

## Checklist
- [x] Tests for new code (see [advice on the tests we use](https://github.com/PostHog/posthog-js#tiers-of-testing))
- [x] Accounted for the impact of any changes across different browsers
- [x] Accounted for backwards compatibility of any changes (no breaking changes in posthog-js!)

### CODE REVIEW

File: src/posthog-surveys.ts [hunk 1]
```diff
@@ -19,6 +19,11 @@ export class PostHogSurveys {
     }
 
     getSurveys(callback: SurveyCallback, forceReload = false) {
+        // Incase we manage to load the surveys script, but config says not to load surveys
```
Comment (jurajmajerik) [inline src/posthog-surveys.ts:22]:
```suggestion
        // In case we manage to load the surveys script, but config says not to load surveys
```

**Review by jurajmajerik — APPROVED** [review]

File: src/__tests__/decide.js [hunk 2]
```diff
@@ -20,6 +20,27 @@ const expectDecodedSendRequest = (send_request, data, noCompression) => {
     })
 }
 
+const checkScriptsForSrc = (src, negate = false) => {
+    const scripts = document.querySelectorAll('body > script')
+    let foundScript = false
+    if (scripts.length > 0) {
```
Comment (pauldambra) [inline src/__tests__/decide.js:26]: nit-picking... do we need the scripts length check?

MDN says `querySelectorAll` always returns a NodeList so it's safe to call length?

Comment (pauldambra) [reply src/__tests__/decide.js:26]: oh, it's a test file in which case that's very nit picky 🤣

Comment (neilkakkar) [reply src/__tests__/decide.js:26]: oh lol, indeed good point I changed what's happening inside this condition later on, it's not needed anymore will remove xD

**Pull request merged**